app/assets/stylesheets/custom.scss

```diff
@@ -2,6 +2,10 @@
   font-family: 'Source Sans Pro', sans-serif;
 }
 
+.btn-success {
+  background-color:  #8BA65F;
+}
```

Comment:
It might be good to use a color variable in this file. see [sass guide.](http://sass-lang.com/guide)